### PR TITLE
COM-19983: Update CSS for blog post

### PR DESCRIPTION
**Pull request by SylvainGuittard**
jira issue: https://jira.ez.no/browse/COM-19983

### CODE REVIEW

**Comment by SylvainGuittard** [comment]
ready for review @damianz5 
(I don't know why I cannot find you in the reviewers...)

File: web/assets/css/blog_post.css [hunk 1]
```diff
@@ -34,15 +34,24 @@
     font-family: 'Open Sans', sans-serif;
 }
 
+.blog-post-body h1{
```
Comment (clash82) [inline web/assets/css/blog_post.css]: `.blog-post-body h1 {`

File: web/assets/css/blog_post.css [hunk 2]
```diff
@@ -34,15 +34,24 @@
...
+    font-size: 43px;
+}
+
+.blog-post-body h2{
```
Comment (clash82) [inline web/assets/css/blog_post.css]: same as https://github.com/ezsystems/ezplatform-com/pull/75/files#r102899965

File: web/assets/css/blog_post.css [hunk 3]
```diff
@@ -34,15 +34,24 @@
...
+    font-size: 32px;
+}
+
+.blog-post-body h3{
```
Comment (clash82) [inline web/assets/css/blog_post.css]: same as https://github.com/ezsystems/ezplatform-com/pull/75/files#r102899965

File: web/assets/css/blog_post.css [hunk 4]
```diff
@@ -34,15 +34,24 @@
...
 .blog-post-body h2,
 .blog-post-body h3,
 .blog-post-body h4 {
-    margin-top: 0;
-}
-
-.blog-post-body h2 {
-    margin-top: 50px;
+    margin-top: 40px;
+    font-weight: 600
```
Comment (clash82) [inline web/assets/css/blog_post.css]: missing semicolon

**Comment by SylvainGuittard** [comment]
@clash82 I found the problem: I pushed the wrong branch... Too many VMs 😄

**Pull request merged**